### ISSUE TITLE
1.19 mouse snapping and can’t scroll when shifting

### ISSUE THREAD

**Issue by Piipperi** [GameParrot/minecraft-mac-window-fix#10]
It appears these fixes are completely broken with 1.19. First it took me a while to even get it to launch with the custom Java executable. After I finally got it to launch with the Gamma-version, I instantly noticed the mouse snapping and that I cannot scroll when holding shift.

For reference, I am using an Intel Mac.

**Comment by GameParrot** [comment]
You will need to re-run the latest installer; I already updated it, but you probably didn't update your copy.

**Comment by CreepyHobo** [comment]
How can I re-run the latest installer?  Previously I entered this command in to terminal:

curl -L 'github.com/GameParrot/minecraft-mac-window-fix/releases/download/1.1/installer.sh' | bash

Then terminal would display the Java executable to enter in to the Minecraft launcher.

**Comment by Piipperi** [comment]
No, the installer is still really buggy and getting Minecraft to launch is difficult, it just doesn't seem to want to open
